editor/src/core/shared/element-path-tree.ts

```diff
@@ -74,7 +74,7 @@ export function reorderTree(
       (elementChild) => {
         switch (elementChild.type) {
           case 'JSX_ELEMENT': {
-            const allChildrenAreElements = elementChild.children.every(isJSXElement)
+            const allChildrenAreElements = elementChild.children.every(isUtopiaElement)
```

Comment:
I am relaxing this check to include not just JSXElement, but JSXFragment and JSXConditionalExpression too. In fact, to include all utopia element-like things that have a uid and can be 1-1 associated with their metadata. (text children, and JsxArbitraryBlocks are still filtered out